test/suite/test_bug010.py

```diff
@@ -37,7 +37,7 @@
 class test_bug010(wttest.WiredTigerTestCase):
     name = 'test_bug010'
     uri = 'table:' + name
-    num_tables = 1000
+    num_tables = 2000 if wttest.islongtest() else 100
```

Comment:
(Again, I'm not sure if doubling the original for the long test adds better testing.)
Running this as a short test is not meaningful. I ran your modified test and the current test against the original fix (changeset `4235c69`) and its unfixed parent (changeset `8ab5d1ce`). (Had to fix up the `set_item` changes for cursor assignment.)

The current test (from develop) fails in the parent and passes in the fixed branch. Your modified/shortened test passes in both. So your short version is no longer testing the bug.

So leave as-is or maybe make the test long-only, but this form is not helpful.

Reply:
I did a similar exercise, and saw that it fails reliably between 120 and 140 for me - so I've moved the lower number to 200.